crates/bevy_math/src/curve/easing.rs

```diff
@@ -84,6 +84,64 @@ where
     }
 }
 
+mod easing_functions {
+    use core::f32::consts::{FRAC_PI_2, FRAC_PI_3, PI};
+
+    use crate::{ops, FloatPow};
+
+    #[inline]
+    pub(crate) fn sine_in(t: f32) -> f32 {
+        1.0 - ops::cos(t * FRAC_PI_2)
+    }
+    #[inline]
+    pub(crate) fn sine_out(t: f32) -> f32 {
+        ops::sin(t * FRAC_PI_2)
+    }
+
+    #[inline]
+    pub(crate) fn back_in(t: f32) -> f32 {
+        let c = 1.70158;
```

Comment:
What is this constant, is it some value (apparently not sqrt(3)) or is it just arbitrary?

Reply:
took it from https://easings.net/#easeInBack, I pretty much copied the "Math function" part